recipes/solaris-gpu/meta.yaml

```diff
@@ -0,0 +1,117 @@
+{% set name = "solaris" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5b357c60c87bbbb79dd8386f94be6432cf205ea7b44b4959a161d476eb67861b
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  entry_points:
+    - geotransform_footprints = solaris.bin.geotransform_footprints:main
+    - make_graphs = solaris.bin.make_graphs:main
+    - make_masks = solaris.bin.make_masks:main
+    - mask_to_polygons = solaris.bin.mask_to_polygons:main
+    - spacenet_eval = solaris.bin.spacenet_eval:main
+    - solaris_run_ml = solaris.bin.solaris_run_ml:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - {{ compiler('c') }}
```

Comment:
Compilers do not go in the host section.

Reply:
Thanks - fixed in the new commit.